### PR TITLE
Playing with the configuration parameters of the FluxPointEstimator and the LightCurveEstimator

### PR DESCRIPTION
**Pull request by bkhelifi**
**Description**
This pull request is associated to the issue #4053. In the associated tutorials, it is precised how to change the important parameters of these two estimators

### CODE REVIEW

File: examples/tutorials/analysis-time/light_curve.py [hunk 1]
```diff
@@ -201,13 +201,18 @@
...
+lc_maker_3d.selection_optional = "all"  # Add the computation of upper limits and likelihood profile
 lc_3d = lc_maker_3d.run(analysis_3d.datasets)
 
 
 ######################################################################
 # The LightCurve object contains a table which we can explore.
 # 
 
+# Example showing how to change just before plotting the threshold on the signal significance (points vs upper limits)
+lc_3d.sqrt_ts_threshold_ul = 16
```
Comment (adonath) [inline examples/tutorials/analysis-time/light_curve.py:215]: Note that this is sqrt(TS), so you effectively apply a threshold of 16 sigma (!) here. I guess you rather mean 4?
```suggestion
lc_3d.sqrt_ts_threshold_ul = 4
```

File: examples/tutorials/starting/analysis_1.py [hunk 2]
```diff
@@ -396,8 +396,19 @@
...
+              "n_sigma": 2, #Number of sigma to use for asymmetric error computation
+              "n_sigma_ul": 3, #Number of sigma to use for upper limit computation
+              }
+analysis.config.flux_points.parameters = config_dict
+
 analysis.get_flux_points()
 
+# Example showing how to change the threshold on the signal significance (points vs upper limits)
+fp = analysis.flux_points.data
+fp.sqrt_ts_threshold_ul = 10
```
Comment (adonath) [inline examples/tutorials/starting/analysis_1.py:411]: Same here it is `sqrt(TS)` so you apply again 10 sigma here...
```suggestion
fp.sqrt_ts_threshold_ul = 3
```

**Review by adonath — APPROVED** [review]
Thanks @bkhelifi, I have left two minor inline comments.

File: examples/tutorials/starting/analysis_1.py (hunk 2 above)
Comment (bkhelifi) [reply examples/tutorials/starting/analysis_1.py:411]: the same here

**Review by adonath — APPROVED** [review]
Thanks @bkhelifi, I have no other comments.

**Pull request merged**